pkg/multus/multus.go

```diff
@@ -536,17 +549,17 @@ func CmdAdd(args *skel.CmdArgs, exec invoke.Exec, kubeClient *k8s.ClientInfo) (c
 		pod, err = kubeClient.GetPod(string(k8sArgs.K8S_POD_NAMESPACE), string(k8sArgs.K8S_POD_NAME))
 		if err != nil {
 			var waitErr error
-			// in case of ServiceUnavailable, retry 10 times with 0.5 sec interval
-			if errors.IsServiceUnavailable(err) {
-				pollDuration := 500 * time.Millisecond
-				pollTimeout := 5 * time.Second
+			// in case of a retriable error, retry 10 times with 0.25 sec interval
+			if isCriticalRequestRetriable(err) {
+				pollDuration := 250 * time.Millisecond
+				pollTimeout := 2500 * time.Millisecond
```

Comment:
optional nit: this could go in a top level constant so it's in the same place as the pollDuration is now, such as: https://github.com/k8snetworkplumbingwg/multus-cni/blob/master/pkg/multus/multus.go#L52-L55

Reply:
Done.